docs/products/kafka/howto/kcat.rst

```diff
@@ -0,0 +1,112 @@
+Use kcat with Aiven for Apache Kafka
+====================================
+
+`kcat <https://github.com/edenhill/kcat>`_ (formerly known as kafkacat) is a generic non-JVM producer and consumer for Apache Kafka. 
+
+It can be used to produce and consume records to Apache Kafka topics as well as to list service configurations.
+
+
+Install kcat
+-------------
+
+kcat is an open source tool available from GitHub at https://github.com/edenhill/kcat. Installation instructions are provided in the repository README.
+
+Retrieve Aiven for Kafka SSL certificate files
+----------------------------------------------
+
+Aiven for Apache Kafka by default enables TLS security. 
+The certificates can be manually downloaded from the service overview page in the Aiven console, or via the :ref:`dedicated Aiven CLI command <avn_service_user_creds_download>`.
+
+Setup a kcat configuration file
+-------------------------------
+
+While kcat accepts all connection configuration parameters in the command line, using a configuration file helps minimising the code needed for any sequent calls.
+
+.. Note:: 
+
+    This example requires kcat **version 1.4.0-RC1** or greater (the ``-F`` option was added after 1.3.1). You can specify the same options on the command line with earlier versions.
+
+A kcat configuration file enabling the connection to an Aiven for Apache Kafka service with TLS security must contain the following entries:
+
+* ``bootstrap.servers``: pointing to the Aiven for Apache Kafka service URI, that can be found in the service overview in Aiven console
```

Comment:
for consistency, let's use same approach for every of these entries. Now part of them start with a verb and others with a noun

Reply:
Fixed, thanks!